### PR TITLE
feat(selection-bar-item): [LIBS-371] display only and clear selection callback 

### PR DESCRIPTION
**Pull request by simonadomnisoru**
Implements [LIBS-371](https://dhis2.atlassian.net/browse/LIBS-371)

---

### Description

Allow clearing of single selections from `SelectionBarItem` component.
Add a `displayOnly` SelectionBarItem component
[Figma prototype](https://www.figma.com/proto/9zvqbQKtrWIElNQ8ZYxyha/capture-context-selector?page-id=0%3A1&node-id=3%3A3435&viewport=1740%2C62%2C0.28&scaling=min-zoom&starting-point-node-id=3%3A1134) 

---

### Checklist

-   [x] API docs are generated
-   [x] Tests were added
-   [x] Storybook demos were added
---

### Screenshots

Screenshot 2022-11-18 at 12 23 47

### CODE REVIEW

**Comment by cooper-joe** [comment]
Pushed a commit to adjust icon and styling. Updated screenshot:
image

**Review by cooper-joe — APPROVED** [review]
Design LGTM 👍

File: components/selector-bar/src/selector-bar-item/selector-bar-item.js [hunk 1]
```diff
@@ -29,32 +29,62 @@ export const SelectorBarItem = ({
...
     const Icon = open ? IconChevronUp24 : IconChevronDown24
 
     return (
         <button
             ref={buttonRef}
-            className={cx('selector-bar-item', className)}
+            className={cx(
+                'selector-bar-item',
+                className,
+                !displayOnly ? 'pointer' : ''
```
Comment (Mohammer5) [inline components/selector-bar/src/selector-bar-item/selector-bar-item.js:49]: Nitpick: The class name `pointer` includes the style that's supposed to be applied. IMO the class name should be descriptive & declarative, so another name would be more appropriate, maybe `openable` (not sure, but I suppose that's the intention here, right?)? 

Then the style applied when the selector bar item is openable can be changed without having to either change the or have a wrong  class name.

This is super nit picky though, so I'll leave that decision up to you

**Review by Mohammer5 — APPROVED** [review]
LGTM code-wise! :tada: 
Just one minor thing that I'll leave up to you

**Pull request merged**